Content.Server/Silicons/Borgs/BorgSystem.MMI.cs

```diff
@@ -73,6 +76,19 @@ private void OnMMILinkedMindAdded(EntityUid uid, MMILinkedComponent component, M
             return;
 
         _mind.TransferTo(mindId, component.LinkedMMI, true, mind: mind);
+
+        BorgJobSwitch((mindId, mind), component);
+    }
+
+    /// <summary>
+    ///     Switches an MMI-d mind's job to borg and stores the previous value so it can be restored
+    /// </summary>
+    private void BorgJobSwitch(Entity<MindComponent> mind, MMILinkedComponent comp)
+    {
+        if(_roles.MindHasRole<JobRoleComponent>(mind.Owner, out var job))
+            comp.OldJob = job.Value.Comp1.JobPrototype;
+
+        _roles.MindAddJobRole(mind, mind, true, "Borg");
```

Comment:
don't hardcode prototypes into code. When you genericize this function make this a ProtoId<t> so its validated properly.